frontend/src/domain/store/pages/Library/index.js

```diff
@@ -31,16 +33,33 @@ function Library() {
     setSliceViewArray(gameList.slice(indexOfFistPost, indexOfLastPost));
   };
 
+  useEffect(() => {
+    setOs(getOsList(viewArray));
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [viewArray]);
+
+  const getOsList = (viewArray) => {
+    console.log('xxxx', viewArray);
```

Comment:
```suggestion
    console.log('xxxx', viewArray);
```
console.log는 머지하기 전에 삭제합시다.